spec/utils/testing.js

```diff
@@ -16,6 +16,7 @@ var ispn = require('../../lib/infinispan');
 var u = require('../../lib/utils');
 var protocols = require('../../lib/protocols');
 
+exports.serverDirName = "infinispan-js-serverng";
```

Comment:
this is not always true. when `run-servers.sh --ci` is run, the server dir is another location.

```
Error: spawn /tmp/infinispan-js-serverng//bin/server.sh ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:232:19)
    at onErrorNT (internal/child_process.js:407:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
```